components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/impl/MLModelHandler.java

```diff
@@ -391,6 +400,13 @@ public String streamingPredict(int tenantId, String userName, long modelId, Stri
             throw new MLModelHandlerException(msg);
         }
 
+        if (!isValidModelStatus(modelId, tenantId, userName)) {
+            String msg = String
+                    .format("This model cannot be used for prediction. Status of the model for model id: %s for tenant: %s and user: %s is not 'Complete'",
```

Comment:
we are using "msg" as a string constant. The first argument string for format function has format specifiers in it. Therefore it was decided to use an inline string as the first argument just as in other places of the code.